ParcelDistributionCenter/ParcelDistributionCenter.Web/Controllers/PackagesController.cs

```diff
@@ -97,6 +127,13 @@ public ActionResult Edit(Package package)
                 return View(nameof(DisplayPackages));
             }
         }
+        // GET: PackagesController/Details/5
+        public ActionResult Details(int packageNumber)
+        {
+            _memoryRepository.LoadData();
+            var model = _packageHandler.FindPackageByNumber(packageNumber);
```

Comment:
Tutaj musi być sprawdzenie, czy paczka nie jest nullem. Jeśli nie jest to widok a, jeśli jest to jakiś widok b.

Reply:
Ok